.github/workflows/tests.yaml

```diff
@@ -0,0 +1,31 @@
+name: CI test
+
+# Trigger the workflow on push and cron
+on:
+  push:
+  schedule:
+  - cron:  '5 0 * * *'
+
+
+jobs:
+  build:
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    steps:
+        - uses: actions/checkout@v2
+        - uses: goanpeca/setup-miniconda@v1
+          with:
+            auto-update-conda: true
+            channels: conda-forge, nodefaults
+        - name: Install prerequisites
+          shell: bash -l {0}
+          run: |
+            conda install pocl islpy pyopencl pytest pip
+            pip install git+https://gitlab.tiker.net/inducer/{loopy,meshmode,dagrt,leap,modepy,meshpy,grudge}
+            pip install -e .
+        - name: Run tests
+          shell: bash -l {0}
+          run: pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw
```

Comment:
Should switch over to onto https://gitlab.tiker.net/inducer/grudge/-/merge_requests/71.

Reply:
`wave-eager.py` looks like this: https://gitlab.tiker.net/inducer/grudge/-/merge_requests/71/diffs#db52a80d4f3415f8822166daad6dec45c3beae60

Reply:
I have completed a first pass to port my mtc/euler branch over onto the new array context stuff.   Can anyone help me test it?  Apparently, I need help setting up my environment to test it.   I think I just need to checkout certain branches of grudge/meshmode?

Reply:
PR #13 is the array-context thing - now separate from the MPI changes.